packages/jmix-react-antd/src/ui/form/controls/RichTextArea.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { JmixFormFieldWrapper } from "./base/JmixFormFieldWrapper";
+import { JmixFormFieldProps } from "./base/JmixFormFieldProps";
+import { Editor, EditorProps } from "react-draft-wysiwyg";
+import styles from "./RichTextArea.module.less";
+import classNames from "classnames";
+import { getMainStore } from "@haulmont/jmix-react-core";
+import { observer } from "mobx-react";
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+
+export const RichTextArea = observer(
+  ({
+    entityName,
+    propertyName,
+    formItemProps,
+    ...rest
+  }: JmixFormFieldProps & EditorProps) => {
+    const mainStore = getMainStore();
+
+    if (!mainStore || !mainStore.locale) {
+      return null;
+    }
+
+    return (
+      <JmixFormFieldWrapper
+        entityName={entityName}
+        propertyName={propertyName}
+        formItemProps={formItemProps}
+        renderField={isReadOnly => (
+          <Editor
```

Comment:
![image](https://user-images.githubusercontent.com/8298981/142593563-61e815e2-5eb0-4435-9bb0-b08268a5a490.png)

It seems that H1 text looks exactly like Normal text.

Reply:
This is a problem of the whole app. Not only this component